build.gradle

```diff
@@ -80,6 +82,7 @@ dependencies {
     testCompile group: 'org.cassandraunit', name: 'cassandra-unit-shaded', version: '3.3.0.2'
     testCompile 'com.datastax.cassandra:cassandra-driver-core:3.6+:tests'
     testCompile 'org.apache.commons:commons-exec:1.3+'
+    testCompile group: 'com.datastax.oss.simulacron', name: 'simulacron-driver-3x', version: '0.8.7'
```

Comment:
IIRC, `simulacron` does not support JMX interaction that Cassandra provides (specifically MBeans), 
which is imperative as part of sidecar implementation as we end up dealing with it a lot.  I am fine with
`simulacron` for now as we don't have any JMX interaction yet in the sidecar, and we can think about JMX interaction when we get there.

Reply:
simulacron is not Cassandra, its just a CQL server thats super light weight and programatic responses. Its also something we can fire up 100 instances with like 20mb of ram. For JMX we will need to use something else yes, but im just testing the CQL connectivity and driver logic